packages/synapse-react-client/src/lib/containers/markdown/MarkdownSynapse.tsx

```diff
@@ -404,7 +405,46 @@ export default class MarkdownSynapse extends React.Component<
         )
       })
       // Render tagName as parent element of the children below
-      return <Tag {...props}>{children}</Tag>
+      switch (Tag) {
```

Comment:
Use MUI components for `p`, `h1`, `h2`, `h3`, `ol`, and `ul` in MarkdownSynapse. Otherwise the rendered markdown text does not visually match non-markdown text shown in the AccessRequiementListV2 component.